Admin panel (#235)
student-hub/acs-upb-mobile: android/fastlane/metadata/android/en-GB/changelogs/10032.txt

> Added

Reviewer: + option to set a custom icon for a website (https://github.com/student-hub/acs-upb-mobile/pull/233)

>     - Basic search page for the app, accessible from the home page

Reviewer: How about "App search functionality, accessible from the home page"? I wouldn't call it basic in the changelog.

>     - Users with admin privileges now have a place in the app where they can respond to requests for editing permissions

Reviewer: Less is more:
```suggestion
    - Admins can now respond to requests for editing permissions directly from the app
```

Improved

>     - Web version looks a lot nicer⭐

Reviewer: It doesn't, not yet. Plus this is a changelog for Play Store so it's not really relevant :)

>     - When pressing 'go to today' in the timetable, you are now notified if you are already seeing the current week

Reviewer: Let's be consistent with the labels in the app:
```suggestion
    - When pressing 'Jump to today' in the timetable, you are now notified if you are already seeing the current week
```

    - Dark mode also applies to the splash screen
    - Did some internal ✨magic✨ to improve the app

Fixed
    - Profile picture is now cropped if it doesn't fit

>     - Bug where you couldn't change your password

Reviewer: + more minor bugs
